www/benr01/js/datetimepicker-init.js

```diff
@@ -0,0 +1,47 @@
+var momentDate;
```

Comment:
vyhodit nepotrebne soubory

Reply:
Tenhle soubor potřebuji k inicializaci rozklikávacích kalendářů. Mám to dát do jednoho souboru ke zbytku JS?